daprdocs/content/en/reference/components-reference/supported-name-resolution/nr-mdns.md

```diff
@@ -0,0 +1,27 @@
+---
+type: docs
+title: "mDNS Name Resolution Provider"
+linkTitle: "mDNS"
+description: Detailed information on the mDNS name resolution component
+---
+
+## Configuration format
+
+Multicast DNS (mDNS) is configured automatically in [self-hosted mode]({{< ref self-hosted >}}) by Dapr. There is no configuration needed to use mDNS as your name resolution provider.
+
+## Behaviour
+
+The component resolves target apps by using the host system's mDNS service. You can learn more about mDNS [here](https://en.wikipedia.org/wiki/Multicast_DNS).
+
+### Troubleshooting
+
+On some enterprise-managed systems, mDNS may be disabled on macOS if a network filter/proxy is configured. Check with your IT department if mDNS is disabled and you are unable to use service invocation locally.
```

Comment:
another example here is that mDNS is simply not implemented, like Azure VNET. It would be good to mention this, if this is the same for AWS and GCP.